src/ctype/CTypeSchema.ts

```diff
@@ -93,6 +93,9 @@ export const CTypeModel = {
       default: 'http://kilt-protocol.org/draft-01/ctype#',
       enum: ['http://kilt-protocol.org/draft-01/ctype#'],
     },
+    title: {
```

Comment:
I have added the title. 

This will need to be updated in the Demo Client, so it takes the value `$id` was meant to have.

Reply:
please make a ticket to support these changes in the demo-cient

Reply:
added